TAF_EasyRest/TestFramework/PageComponents/AdministratorPanelComponents/AcceptedCardPageComponent.cs

```diff
@@ -0,0 +1,35 @@
+﻿
+namespace TestFramework.PageComponents.AdministratorPanelComponents
+{
+    public class AcceptedCardPageComponent
+    {
+        IWebDriver driver { get; }
+
+        public AcceptedCardPageComponent(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        private IWebElement _selectWaiterRadioButton => driver.FindElement(By.XPath("(//input)[1]"));
+        private IWebElement _assignButton => driver.FindElement(By.XPath("(//span[text()='Assign'])[1]"));
+
+        public AcceptedCardPageComponent SelectAnyWaiter()
+        {
+            _selectWaiterRadioButton.Click();
+            return this;
+        }
+
+        //method to check that the order was assigned by checking if the number of orders decreased in the current tab
+        public bool ClickAssignButton()
```

Comment:
I agree with Anton, the method name should correspond to its implementation.
But it would be better to split that method into two separate methods:
1) `public AcceptedCardPageComponent ClickAssignButton() { }`
2) `public bool CheckThatCountOfAcceptedOrdersHaveIncreasedByOne() {  }`

Reply:
done